lib/diff.ml

```diff
@@ -17,11 +17,20 @@ type module_ = {
   mdiff : (module_declaration, module_modification) t;
 }
 
+and modtype = {
+  mtname : string;
+  mtdiff : (modtype_declaration, module_modification) t;
+}
+
 and module_modification = Unsupported | Supported of sig_item list
-and sig_item = Value of value | Module of module_
+and sig_item = Value of value | Module of module_ | Modtype of modtype
+
+type item_type = Value_item | Module_item | Modtype_item [@@deriving ord]
 
-type item_type = Value_item | Module_item [@@deriving ord]
-type sig_items = Val of value_description | Mod of module_declaration
+type sig_items =
+  | Val of value_description
+  | Mod of module_declaration
+  | ModType of modtype_declaration
```

Comment:
We use snake case in OCaml, not camel case (weirdly enough!). Could you change that to `Modtype` or `Mod_type`. I'd like the former better as it matches what you used for the `item_type` variant.